### PR TITLE
Enable VSCode debugger

### PR DESCRIPTION
**Pull request by azaslavsky**
This change saves settings and instructions for easily using the built-in VSCode node.js debugger to step through and break on a running local instance of eng-pipes.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```
Comment (chadwhitacre) [inline .vscode/launch.json:2]: Are comments supported here? Usually don't exist in json afaik. 🐭

Comment (azaslavsky) [reply .vscode/launch.json:2]: VSCode configuing `.json` files (`launch.json`, `settings.json`, etc) actually use `jsonc`, which is basically JSON that allows single-line comments and trailing commas to make configuration via JSON a little less maddening.

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                "isDefault": true
+            },
+            "presentation": {
+                "reveal": "silent",
+                "panel": "shared"
+            },
+            "problemMatcher": []
+        }
+    ]
+  }
```
Comment (hubertdeng123) [inline .vscode/tasks.json:21]:
```suggestion
  }
  
```

**Review by hubertdeng123 — APPROVED** [review]
nice add!

**Pull request merged**